src/utils/file/index.ts

```diff
@@ -578,3 +579,20 @@ export function getLatestVersionFiles(files: EnteFile[]) {
         (file) => !file.isDeleted
     );
 }
+
+export function getUserPersonalFiles(files: EnteFile[]) {
+    const user: User = getData(LS_KEYS.USER);
+    if (!user?.id) {
+        throw Error('user missing');
+    }
+    return files.filter((file) => file.ownerID === user.id);
+}
+
+export const getTotalFileCount = async () => {
```

Comment:
Also, will this method return count of collection_files or files? I am not sure what do we need it return :)

Reply:
It will return the count of `collection_files`

Reply:
as, Files are not deduped during export, so showing `collection_files` count made more sense